src/lib/server/eliommod_cookies.ml

```diff
@@ -274,7 +274,9 @@ let compute_session_cookies_to_send
         | Eliom_common.SCData_session_expired -> None
         | Eliom_common.SC c ->
             let v = c.Eliom_common.sc_set_value in
+(*
             c.Eliom_common.sc_set_value <- `Used;
```

Comment:
Reverted since the tab cookies to send can be computed twice: once for the headers, once for the HTML page (to set a JavaScript variable)